components/icon-buttons/Web/Web-icon-buttons.md

```diff
@@ -8,227 +8,167 @@ path: /components/buttons
+   <i class="material-icons mdc-icon-button__icon">favorite_border</i>
+</button>
+```
 
-<img src="assets/web-toggle-icon-button.png" alt="4 images arranged in a 2 by 2 array, each image with a favorite icon in the upper-right corner" width="50%">
-To generate the example with background images of your choosing, you will need containers for your tiled images, and containers to resize your images. You may also need to change the icon color to ensure that the icon is visible on your image (see the `index.scss` file for the tiled container \[`container`\] and the image container \[`image-with-fav`\] and the icon button color \[`favve-button`\]).
+Then, instantiate an `MDCIconButtonToggle` on the root element.
 
-* In your `*.js` file,
-    1. Import `MDCIconButtonToggle`:
-        ```js
-        import {MDCIconButtonToggle} from '@material/icon-button';
-        ```
-    1. Instantiate the icon button toggle for each button:
-        ```js
-		const iconToggle0 = new MDCIconButtonToggle(document.querySelector('.btn1'));
-		iconToggle0.unbounded = true;
+```js
+import {MDCIconButtonToggle} from '@material/icon-button';
+const iconToggle = new MDCIconButtonToggle(document.querySelector('.mdc-icon-button'));
+```
 
-		const iconToggle1 = new MDCIconButtonToggle(document.querySelector('.btn2'));
-		iconToggle1.unbounded = true;
-	```
-* In your `*.html` file, you can now add the iterations of the following, which shows containers with background images:
-    ```html
-    <div class="container">
-          <div class="image-with-fave">
-            <img src="https://cdn.glitch.com/4f0b2993-dd70-46bf-916f-a6e81af58957%2Fimage1.jpeg?v=1570131499813" />
-            <button
-              id="bookmark-0"
-              class="mdc-icon-button btn1 fave-button"
-              aria-label="Add to favorites"
-              aria-hidden="true"
-              aria-pressed="false"
-            >
-              <i
-                class="material-icons mdc-icon-button__icon mdc-icon-button__icon--on"
-                >favorite</i
-              >
-              <i class="material-icons mdc-icon-button__icon">favorite_border</i>
-            </button>
-          </div>
-          <div class="image-with-fave">
-            <img
-          src="https://cdn.glitch.com/4f0b2993-dd70-46bf-916f-a6e81af58957%2Fimage2.jpeg?v=1570131499876" />
-            <button
-              id="bookmark-1"
-              class="mdc-icon-button btn2 fave-button"
-              aria-label="Add to favorites"
-              aria-hidden="true"
-              aria-pressed="false"
-            >
-              <i
-                class="material-icons mdc-icon-button__icon mdc-icon-button__icon--on"
-                >favorite</i>
-              <i class="material-icons mdc-icon-button__icon">favorite_border</i>
-              </button>
-          </div>
-          ...
-        </div>
-    ```
+### Icon Button Toggle with SVG
```

Comment:
Titles need to follow sentence case: capitalize the first word, lower-case all following words unless they're acronyms or proper nouns.